components/BuildPage/WaysToGrow.jsx

```diff
@@ -0,0 +1,130 @@
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { Card, CardTitle } from 'components/ui/card';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useState } from 'react';
+import Content from './Content';
+
+const ways = [
+  {
+    title: 'Earn Dev Rewards by building on Olas Protocol',
+    imageSrc: '/images/build-page/earn-dev-rewards.jpg',
+    description: (
+      <>
+        <p className="mb-4">
+          Contribute valuable code units — like agents or components — to the
+          Olas protocol and have a chance at receiving Developer Rewards.
+        </p>
+        <p className="mb-4">
+          Dev Rewards is a part of the protocol that facilitates the
+          distribution of capital to developers who contribute to various
+          services in the ecosystem.
+        </p>
+        <p>
+          This system is designed to reward both the contribution of code
+          components and entire agents.
+        </p>
+      </>
+    ),
+    linkAction: (setDevRewards) => (
+      <a
+        className="text-purple-600 cursor-pointer"
+        onClick={() => setDevRewards(true)}
```

Comment:
is this correct, in figma it says to open `https://olas.network/dev-rewards`?

Reply:
`https://olas.network/dev-rewards` previously lead to a section on the build page that no longer exists. Roman instead asked to do a temp modal that would display that section...